.eslintignore

```diff
@@ -2,3 +2,6 @@ dist
 node_modules
 packages/manager/tools/sao-ovh-manager-module/template/src
 packages/manager/tools/sao-ovh-manager-app/template
+packages/manager/apps/container
```

Comment:
**question:** Any particular reason to ignore ESLint on `container`?

Reply:
I think it was to avoid having too many changes in the PR and diluting the features added (the generator).
We can fix the eslint errors in another PR or in this one if you prefer.